src/i18n/types.ts

```diff
@@ -27,6 +28,8 @@ export type WalletMessages = {
   'wallet.column.value': string
   'wallet.action.send': string
   'wallet.action.receive': string
+  'wallet.imports.phrase': string
+  'wallet.imports.enter.phrase': string
```

Comment:
[nitpick] `wallet.imports.enterphrase` (`enterphrase` instead of `enter.phrase`)